langroid/language_models/openai_gpt.py

```diff
@@ -1622,18 +1676,30 @@ def _prep_chat_completion(
                 )
             )
         if tools is not None:
+            if self.config.parallel_tool_calls is not None:
+                args["parallel_tool_calls"] = self.config.parallel_tool_calls
+
+            if any(t.strict for t in tools) and (
+                self.config.parallel_tool_calls is None
```

Comment:
simplify - just need:
```
any(...) and self.config.parallel_tool_calls
```

Reply:
Doesn't parallel_tool_calls default to `True`? I think we need to catch the `None` case

Reply:
you're right!